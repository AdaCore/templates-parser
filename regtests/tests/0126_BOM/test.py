from test_support import *

run('testme', ["with_bom.tmplt"])
run('print_tree', ["with_bom.tmplt"])
