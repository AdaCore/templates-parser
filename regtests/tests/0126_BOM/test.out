ONE is properly set
[INFO] with_bom.tmplt
[C_INFO]  1 FALSE
[TEXT] U+<FEFF>
[IF] (@_ONE_@ = 1)
[TEXT] ONE is properly set
[ELSE]
[TEXT] ONE is not OK
[END_IF]
